dist/info.js

```diff
@@ -0,0 +1,63 @@
+// import data object {"caveat": {}, "veneer": {}}
+import { projects, list } from "./data";
+
+const currentUrl = new URL(window.location.href);
+const path = window.location.pathname;
```

Comment:
all calculations need to do on "DOMContentLoaded" event
so, you should put all code, that **runs** outside, inside event handler

Reply:
ok